packages/plugin-ext/src/plugin/plugin-manager.ts

```diff
@@ -274,11 +275,11 @@ export class PluginManagerExtImpl implements PluginManagerExt, PluginManager {
         const asAbsolutePath = (relativePath: string): string => join(plugin.pluginFolder, relativePath);
         const logPath = join(configStorage.hostLogPath, plugin.model.id); // todo check format
         const storagePath = join(configStorage.hostStoragePath || '', plugin.model.id);
-        async function defaultGlobalStorage(): Promise<string> {
-            const globalStorage = join(os.homedir(), '.theia', 'globalStorage');
-            await fs.ensureDir(globalStorage);
-            return globalStorage;
-        }
+        const defaultGlobalStorage = async () => {
```

Comment:
I wonder can we make `hostGlobalStoragePath` mandatory to avoid introducing env server ext apI?

Reply:
Yes, this would be better.